scripts/install-snapshot.sh

```diff
@@ -77,6 +79,29 @@ function delete_snapshot_crd() {
     kubectl delete -f "${VOLUME_SNAPSHOT}" --ignore-not-found
 }
 
+# parse the kubernetes version
+# v1.17.2 -> kube_version 1 -> 1  (Major)
+# v1.17.2 -> kube_version 2 -> 17 (Minor)
+function kube_version() {
+    echo "${KUBE_VERSION}" | sed 's/^v//' | cut -d'.' -f"${1}"
+}
+
+KUBE_VERSION=$(kubectl version --short 2>/dev/null | grep "^Server Version" | cut -d' ' -f3)
```

Comment:
Redirecting stderr to /dev/null might make debugging more difficult in case an error occurs. Is there a good reason to not display potential errors here?

Reply:
Because we do not want the next line (condition) to return false in case of error.

Reply:
But the output of stderr is not captured in `KUBE_VERSION` if you do not use `2>&1` or a similar redirection?

Reply:
fixed now. Thanks Neils.